src/time_tracker/core.clj

```diff
@@ -25,21 +25,6 @@
        (log/info {:event ::server-stop})
        (teardown!)))))
 
-(defonce server-stop-fn (atom nil))
-
-(defn repl-start-server! []
-  (let [stop-fn (start-server!)]
-    (swap! server-stop-fn (constantly stop-fn))))
-
-(defn repl-stop-server! []
-  (when @server-stop-fn
-    (@server-stop-fn)
-    (reset! server-stop-fn nil)))
-
-(defn repl-restart-server! []
-  (repl-stop-server!)
-  (repl-start-server!))
-
 (defn- start-server-as-gid!
```

Comment:
If we're moving REPL related functions around, this function should go too. This function is really only for REPL testing and nothing else.